test/e2e/medusa_test.go

```diff
@@ -39,6 +40,16 @@ func createSingleMedusaJob(t *testing.T, ctx context.Context, namespace string,
 	verifyBackupJobFinished(t, ctx, f, dcKey, backupKey)
 	restoreBackupJob(t, ctx, namespace, f, dcKey)
 	verifyRestoreJobFinished(t, ctx, f, dcKey, backupKey)
+
+	// Scale the cluster to verify that the previous restore won't break the new pod
+	t.Log("Scaling the cluster to 3 nodes")
+	err = f.Get(ctx, kcKey, kc)
```

Comment:
suggestion: For future, if you wish to type less:

``require.NoError(f.Get(ctx, kcKey, kc))``